src/App.tsx

```diff
@@ -22,6 +67,7 @@ export const App: React.FC = () => {
         <header className="todoapp__header">
```

Comment:
split it into components (Header, Footer, TodoList, TodoItem, ErrorNotification and so on)
<img width="772" alt="image" src="https://user-images.githubusercontent.com/102411130/209461736-284d855a-f9ec-4645-bf66-22ed42b588ad.png">
requirements